### PR TITLE
Mitigate toggle service DOS risk

### PR DESCRIPTION
**Pull request by mukama**
Resolves https://github.com/bbc/simorgh-infrastructure/issues/1323

**Overall change:**
A risk was identified during threat modelling where the 3 second timeout for retrieving toggles posed a DOS risk.
- In the event of an outage of the toggles api, this PR reduces the timeouts when requesting toggle values to 1 second for at least 60 seconds after which the timeout reverts back to 3 seconds.
- Increases the length of the in memory cache, to allow the api to be unavailable for much longer.

**Code changes:**
- Increases cache size via `SIMORGH_CONFIG_CACHE_ITEMS` from 100 to 200 items
- Adds custom timeout for toggles api outages

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [x] I have assigned this PR to the Simorgh project
- [ ] (BBC contributors only) This PR follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false yarn test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

File: envConfig/live.env [hunk 1]
```diff
@@ -11,9 +11,11 @@ LOG_DIR='/var/log/simorgh'
 LOG_LEVEL='info'
 LOG_TO_CONSOLE=false
 SIMORGH_APP_ENV=live
-SIMORGH_CONFIG_CACHE_ITEMS=100
+SIMORGH_CONFIG_CACHE_ITEMS=200
```
Comment (andrewscfc) [inline envConfig/live.env:14]: Why has this value been changed? We will be caching no more items with this chnage to my knowledge?

**Review by andrewscfc — COMMENTED** [review]
I'm a bit wary of the complexity in this PR we may be able to make a change as simple as reducing the `SIMORGH_CONFIG_TIMEOUT_SECONDS` to `1` from `3`? Could we maybe do an exploratory PR to add some logging to see how long on average config requests take to come back? If we find they are reliably below a second, we could just make the simple change above; if not we could look to make a more complex change.

**Comment by mukama** [comment]
> I'm a bit wary of the complexity in this PR we may be able to make a change as simple as reducing the `SIMORGH_CONFIG_TIMEOUT_SECONDS` to `1` from `3`? Could we maybe do an exploratory PR to add some logging to see how long on average config requests take to come back? If we find they are reliably below a second, we could just make the simple change above; if not we could look to make a more complex change.

Great idea @andrewscfc. I'll have a PR out to log the request time 👍

**Comment by mukama** [comment]
We created an exploratory PR #1382 to find out how long an average toggle service request takes to respond. From the cloud watch logs, when we checked the toggle_api_response_time event, we observed an average ~0.056 seconds response time on toggle service requests.

As a result, this DOS risk can be safely mitigated by reducing the `SIMORGH_CONFIG_TIMEOUT_SECONDS` from 3 seconds to 1 second.

**Pull request closed without merging**